com.unity.render-pipelines.universal/ShaderLibrary/ShaderVariablesFunctions.hlsl

```diff
@@ -219,18 +238,41 @@ half3 MixFogColor(real3 fragColor, real3 fogColor, real fogFactor)
     return fragColor;
 }
 
-half3 MixFog(real3 fragColor, real fogFactor)
+float3 MixFogColor(float3 fragColor, float3 fogColor, float fogFactor)
+{
+    #if defined(FOG_LINEAR) || defined(FOG_EXP) || defined(FOG_EXP2)
+        real fogIntensity = ComputeFogIntensity(fogFactor);
```

Comment:
real?

Reply:
Well spotted! 
Fixed in 7dae270899de40b0207c34ea9792327817788c18